i18n/de_DE/messages/messages_laika.json

```diff
@@ -126,15 +126,15 @@
   "donation_form_country_option_LI": "Liechtenstein",
   "donation_form_country_option_LU": "Luxemburg",
   "donation_form_custom_placeholder": "Wunschbetrag, z.B. 23,42",
-  "donation_form_dob_error": "Könnten Sie bitte Ihr Geburtsdatum überprüfen? Irgendetwas scheint da nicht zu stimmen.",
+  "donation_form_dob_error": "Bitte überprüfen Sie das Geburtsdatum. Es muss im Format Tag-Monat-Jahr angegeben werden.",
```

Comment:
Das Format ist eher Tag.Monat.Jahr oder? Finde die Bindestriche verwirrend.

Reply:
Vielleicht besser ein Beispiel angeben, z.B. 29.02.1989?

Reply:
I changed to Tag.Monat.Jahr. An  example is already in the placeholder in the field.

Reply:
I'm amazed this even works, because we're not validating the date format and relying on the generosity of PHP. I've created a ticket for that. https://phabricator.wikimedia.org/tag/wmde-fundraising-tech/

And while I tested the date, I discovered it's broken for laika: https://phabricator.wikimedia.org/T237366